tests/lint/unittest_lint.py

```diff
@@ -525,6 +525,158 @@ def test_load_plugin_command_line() -> None:
     sys.path.remove(dummy_plugin_path)
 
 
+@pytest.mark.usefixtures("pop_pylintrc")
+def test_load_plugin_command_line_and_config_with_inithook_in_config_bad_value() -> None:
```

Comment:
I think these are named incorrectly? Here both settings are in the init? Perhaps add a docstring?

Reply:
I'll add a docstring, but this test is just to ensure that the specific case where the `--load-plugins` CLI argument **and** the `load-plugins` config setting are both provided and contain the same (bad) plugin.